snuba/datasets/__init__.py

```diff
@@ -13,18 +13,18 @@ class Dataset(object):
     This is the the initial boilerplate. schema and processor will come.
     """
 
-    def __init__(self, schema, *, processor,
+    def __init__(self, dataset_schemas, *, processor,
             default_topic: str,
             default_replacement_topic: Optional[str] = None,
             default_commit_log_topic: Optional[str] = None):
-        self._schema = schema
+        self.__dataset_schemas = dataset_schemas
         self.__processor = processor
         self.__default_topic = default_topic
         self.__default_replacement_topic = default_replacement_topic
         self.__default_commit_log_topic = default_commit_log_topic
 
-    def get_schema(self):
-        return self._schema
+    def get_dataset_schemas(self):
```

Comment:
I would add type hints here, just tactically because people right now are used to schema being an instance of the TableSchema class.